x/posts/client/rest/query.go

```diff
@@ -31,6 +31,7 @@ func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/posts/{postID}", queryPostHandlerFn(cliCtx)).Methods("GET")
 	r.HandleFunc("/posts", queryPostsWithParameterHandlerFn(cliCtx)).Methods("GET")
 	r.HandleFunc("/posts/{postID}/poll-answers", queryPostPollAnswersHandlerFn(cliCtx)).Methods("GET")
+	r.HandleFunc("/reactions", queryRegisteredReactions(cliCtx)).Methods("GET")
```

Comment:
Probably to make it more similar to the CLI command, we should change this to 

```suggestion
	r.HandleFunc("/posts/registeredReactions", queryRegisteredReactions(cliCtx)).Methods("GET")
```